src/services/hasura/hasura.service.ts

```
@@ -62,242 +62,285 @@ export class HasuraService {
+				// Special handling for annualIncome
+				if (filterKey === 'annualIncome') {
+					return handleAnnualIncome(cleanFilterValue, cleanConditionValues);
+				}
+
+				switch (condition) {
+					case 'in':
+					case 'contains':
+					case 'includes':
+						return cleanConditionValues.some((value) => {
+							const exactMatch = value === cleanFilterValue;
+							const valueIncludes = value.includes(cleanFilterValue);
```

ESLint

[error] Replace `·` with `⏎↹↹↹↹↹↹↹↹`

(prettier/prettier)